src/trajectory_data_handling/trajectory_sql.hpp

```diff
@@ -5,6 +5,7 @@
 #include <sqlite3.h>
 #include "../data/trajectory_structure.hpp"
 #include "sqlite_querying.hpp"
+#include<tuple>
```

Comment:
```suggestion
#include <tuple>
```
Sæt den over de includes med quotes "", under sqlite3